bims/serializers/location_site_detail_serializer.py

```diff
@@ -35,6 +36,33 @@ class Meta:
             'location_context_document_json',
         ]
 
+
+    def get_site_detail_info(self, obj):
+        site_coordinates = "{latitude}, {longitude}".format(
+            latitude=round(obj.geometry_point.x, 3),
+            longitude=round(obj.geometry_point.y, 3))
+        try:
+            print "Hello World"
```

Comment:
Please remove this. 
I will merge this one, and for future don't print anything.